app/javascript/v3/components/Form/Button.vue

```diff
@@ -56,10 +57,11 @@ const props = defineProps({
 const attrs = useAttrs();
 
 const baseClasses = {
-  outline: 'outline outline-1 -outline-offset-1 focus:ring focus:ring-offset-1',
-  ghost: 'hover:text-600 active:text-600 focus:ring focus:ring-offset-1',
+  outline:
```

Comment:
@muhsin-k Do we need to focus on the outline variant button? The border is handled by outline.